app/models/reviewable_queued_post.rb

```diff
@@ -148,9 +156,12 @@ def perform_delete_user_block(performed_by, args)
   private
 
   def delete_user(performed_by, delete_options)
-    reviewable_ids = Reviewable.where(created_by: created_by).pluck(:id)
-    UserDestroyer.new(performed_by).destroy(created_by, delete_options)
-    create_result(:success) { |r| r.remove_reviewable_ids = reviewable_ids }
+    reviewable_ids = Reviewable.where(created_by: target_created_by).pluck(:id)
```

Comment:
We are passing `target_created_by`, but still using `created_by`

Reply:
Yes.  UserDestroyer still deletes reviewables created by the `target_created_by` user.  The original intent was to get ids of these reviewables so they can be removed from the UI since they'd have been deleted anyway.  What's needed here is reviewables created by the  `target_created_by` user not reviewable with targets they created.